src/main/proto/management.proto

```diff
@@ -131,6 +136,14 @@ message GetCompositeIndicesByVertexLabelRequest {
     VertexLabel vertexLabel = 2;
 }
 
+message GetCompositeIndicesForVertexRequest {
```

Comment:
You already can do this by calling first for all vertices and than checkout all indices. 

My problem with this currently indices are binded to a vertex. If we want to add this we should return a tuple of vertex and index.

Reply:
There are following problems associated with that:

1. If an Index was created without any `label` to it, we can't really retrieve that in any way. Only the index which has `indexOnly` param passed during creation could be retrieved. 
2: For example the index `name` in `GraphOfTheGods` can't be retrieved.
3. We will also need to enable user to create index w/o label `indexOnly` param. `EnsureCompositeIndexForLabel` will always attach to a vertex label.
4. Instead of working on tuple for Vertex and Index, how about we add property to vertex itself? So if an index was created without `indexOnly` its label will be `ALL`. 
5. And since we don't get `label constraint` while doing `management.getGraphIndexes(Vertex.class)` we can set the label property as `UNDEFINED` ? Or maybe a bit too complicated?

What do you think?

Reply:
I think we should handle indices with/without indexOnly as different types in grpc, so there is no confuse. For me currently, the management can be confusing, because if you activate schema constraints so things works differently.